psi4/src/psi4/ccdensity/file_build.cc

```diff
@@ -102,18 +102,33 @@ int file_build(dpdfile4 *File, int inputfile, double tolerance,
               core_left = memoryd - row_length;
 
               /* Make room for another bucket */
-              bucket_offset = (int **) realloc((void *) bucket_offset,
-                                               nbuckets * sizeof(int *));
```

Comment:
We are a bit hit or miss in catching `alloc`, `realloc`, `calloc`, etc calls. However, this one is particularly large so probably a good idea to do so. For each of these can you include explicit bracers in the if/else calls.